docs/HowTo/Configure/Consensus-Protocols/QBFT.md

```diff
@@ -0,0 +1,187 @@
+        "9f66f8a0f0a6537e4a36aa1799673ea7ae97a166": {
+            "balance": "0x446c3b15f9926687d2c40534fdb564000000000000"
+        },
+        "a7f25969fb6f3d5ac09a88862c90b5ff664557a7": {
+            "balance": "0x446c3b15f9926687d2c40534fdb564000000000000"
+        },
+        "f4bbfd32c11c9d63e9b4c77bb225810f840342df": {
+            "balance": "0x446c3b15f9926687d2c40534fdb564000000000000"
+        }
+    },
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+    }
+    ```
+
+The QBFT properties are:
+
+* `blockperiodseconds` - The minimum block time, in seconds.
+* `epochlength` - The number of blocks after which to reset all votes.
+* `requesttimeoutseconds` - The timeout for each consensus round before a round change, in seconds.
+* `blockreward` - Optional reward amount in Wei to reward the beneficiary. Defaults to zero (0).
+    Can be specified as a hexadecimal (with 0x prefix) or decimal string value. If set, then all
+    nodes on the network must use the identical value.
+* `miningbeneficiary` - Optional beneficiary of the `blockreward`. Defaults to the validator
+    that proposes the block. If set, then all nodes on the network must use the same beneficiary.
+* `extraData` - `RLP([32 bytes Vanity, List<Validators>, No Vote, Round=Int(0), 0 Seals])`.
+
+!!! caution
+
+    The `blockperiodseconds`, `blockreward`, and  `miningbeneficiary` properties
+    cannot be updated once your network is started.
+
+    We do not recommend changing `epochlength` in a running network. Changing the `epochlength`
+    after genesis can result in illegal blocks.
+
+The properties with specific values in the QBFT genesis files are:
+
+* `difficulty` - `0x1`
+* `mixHash` - `0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365` for Istanbul
+  block identification.
+
+To start a node on a QBFT private network, use the
+[`--genesis-file`](../../../Reference/CLI/CLI-Syntax.md#genesis-file) option to specify the custom
+genesis file.
+
+### Extra data
+
+The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization
+scheme used in Ethereum. To generate the `extraData` RLP string for inclusion in the genesis file,
+use the [`rlp encode`](../../../Reference/CLI/CLI-Subcommands.md#rlp) Besu subcommand.
+
+!!! example
+
+    ```bash
+    besu rlp encode --from=toEncode.json --type=QBFT_EXTRA_DATA
+    ```
+
+Where the `toEncode.json` file contains a list of the initial validators, in ascending order. To
+write the validator address and copy it to the `toEncode.json` file, use the
+[`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) Besu
+subcommand. For example:
+
+!!! example "One Initial Validator in `toEncode.json` file"
+
+    ```json
+    [
+     "9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb"
+    ]
+    ```
+
+Copy the RLP encoded data to the `extraData` property in the genesis file.
```

Comment:
Can you add the RLP output for this example?

Reply:
Done